common/rtc.c

```diff
@@ -572,169 +573,173 @@ uint8_t RTC_timer_todo = 0;
 uint8_t RTC_timer_done = 0;
 static uint8_t RTC_timer_time[RTC_TIMERS];
 #if defined(MASTER_CONFIG_H)
-    static uint8_t RTC_next_compare;
+static uint8_t RTC_next_compare;
 #endif
 
-void RTC_timer_set(uint8_t timer_id, uint8_t time) {
-    uint8_t t2,i,next,dif;
-    // next is uninitialized, it is correct
-    
-    cli();
-    RTC_timer_todo |= _BV(timer_id);
-	RTC_timer_time[timer_id-1]=time;
-    t2=TCNT2;
-    dif=255;
-    for (i=0;i<RTC_TIMERS;i++) {
-        if ((RTC_timer_todo&(2<<i))) {
-            if ((RTC_timer_time[i]-t2)<=dif) {
-                next = RTC_timer_time[i];
-                dif = next-t2;
-            }
-        }
-    }
-    #if defined(MASTER_CONFIG_H)
-        RTC_next_compare = next;
-    #else
-        if (OCR2A != next) {
-            while (ASSR & (1<<OCR2UB)) {;} // ATmega169 datasheet chapter 17.8.1
-            OCR2A = next;
-        }
-    #endif
-    sei();
-    #if ! defined(MASTER_CONFIG_H)
-        TIMSK2 |= (1<<OCIE2A); // enable interupt again
-    #endif
+void RTC_timer_set(uint8_t timer_id, uint8_t time)
+{
+	uint8_t t2, i, next, dif;
+
+	// next is uninitialized, it is correct
+
+	cli();
+	RTC_timer_todo |= _BV(timer_id);
+	RTC_timer_time[timer_id - 1] = time;
+	t2 = TCNT2;
+	dif = 255;
+	for (i = 0; i < RTC_TIMERS; i++) {
+		if ((RTC_timer_todo & (2 << i))) {
+			if ((RTC_timer_time[i] - t2) <= dif) {
+				next = RTC_timer_time[i];
+				dif = next - t2;
+			}
+		}
+	}
+#if defined(MASTER_CONFIG_H)
+	RTC_next_compare = next;
+#else
+	if (OCR2A != next) {
+		while (ASSR & (1 << OCR2UB));   // ATmega169 datasheet chapter 17.8.1

```

Comment:
Although I agree the formatting was improved in general, the curly brace here was for a reason, to make it more obvious that the loop does nothing in the body. Removing it makes it less obvious. Can we have at least the semicolon on a separate line?

Reply:
You're completely right, the braces should be there and in the mean time I learned that having more braces around things makes the code much more readable. I created a new uncrustify.conf, which you can find [here](https://github.com/dlemper/OpenHR20/blob/086c842efa846b47f1f3efd3417578c6d1cd6986/uncrustify.conf), if you want to have a look